src/main/java/cz/brmlab/yodaqa/analysis/rdf/PropertyGloVeScoring.java

```diff
@@ -0,0 +1,31 @@
+package cz.brmlab.yodaqa.analysis.rdf;
+
+import cz.brmlab.yodaqa.provider.glove.MbWeights;
+import cz.brmlab.yodaqa.provider.glove.Relatedness;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by silvicek on 8/18/15.
+ */
```

Comment:
Can we please have a real class description here. Something general about the scoring method, possibly reference to the sentence-selection repo or some other detailed documentation.